<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>921</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:qlineargradient(spread:pad, x1:1, y1:0.982409, x2:1, y2:0, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255))}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Insert New Staff</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>581</width>
      <height>621</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>First Name</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Last Name</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Username</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Password</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Staff Type</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="addStaff">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>750</y>
      <width>581</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Staff</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backTo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
